### PR TITLE
PAYARA-3758 Cleanup rest service

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins test please

File: nucleus/admin/rest/rest-service/src/main/java/org/glassfish/admin/rest/resources/TemplateCommandPostResource.java [hunk 1]
```diff
@@ -38,7 +38,7 @@
  * holder.
  */
 
-// Portions Copyright [2016-2018] [Payara Foundation]
+// Portions Copyright [2016-2010] [Payara Foundation and/or affiliates]
```
Comment (jGauravGupta) [inline nucleus/admin/rest/rest-service/src/main/java/org/glassfish/admin/rest/resources/TemplateCommandPostResource.java]: Copyright year typo

File: nucleus/admin/rest/rest-service/src/main/java/org/glassfish/admin/rest/logviewer/LineEndNormalizingWriter.java [hunk 2]
```diff
@@ -37,6 +37,7 @@
  * only if the new code is made subject to such option by the copyright
  * holder.
  */
+// Portions Copyright [2019[ Payara Foundation and/or affiliates
```
Comment (jGauravGupta) [inline nucleus/admin/rest/rest-service/src/main/java/org/glassfish/admin/rest/logviewer/LineEndNormalizingWriter.java]: Minor typo in Portions Copyright

**Comment by mulderbaba** [comment]
@Cousjava awesome spring cleanup!

**Comment by Cousjava** [comment]
Jenkins test please

**Pull request merged**